### PR TITLE
Store api general updates

### PR DESCRIPTION
**Pull request by sushmangupta**
Changes made are w.r.t textual, passive voice, grammatical errors, and contextual errors.

### CODE REVIEW

**Review by Christian-Rades — CHANGES_REQUESTED** [review]

File: docs/guides/faq.md [hunk 1]
```diff
@@ -12,24 +12,24 @@
 > }
 > ```
 
-**How can I check whether I'm currently logged in?**
+**How can I check whether I am currently logged in?**
```
[the remark this answers is not in the document]
Comment (sushmangupta) [reply docs/guides/faq.md:15]: Here the auxiliary verb is missing in the sentence:  How can I check whether the user is currently logged in?

File: docs/guides/faq.md [hunk 2]
```diff
@@ -12,24 +12,24 @@
...
 
-> The least you need in order to have a logged in is a `sw-context-token` header. In order to check whether you're actually logged in call the following endpoint
+> The least you need in order to have logged in is, a `sw-context-token` header. In order to check whether you are actually logged in call the following endpoint
```
[the remark this answers is not in the document]
Comment (sushmangupta) [reply docs/guides/faq.md:17]: Suggesting to keep the sentence as follows: 
The least you need to get logged in is an `sw-context-token` header. In order to check whether the user is actually logged in, call the following endpoint

File: docs/guides/faq.md [hunk 3]
```diff
@@ -12,24 +12,24 @@
...
 
 **How can I fetch SEO URLs for requested categories or products**
 
-> When you request includes the `sw-include-seo-urls: 1` header, Shopware will automatically try to resolve relative URLs for all your categories and products. These will then be accessible in the `seoUrls` field of each record.  
\ No newline at end of file
+> When you request include the `sw-include-seo-urls: 1` header. Shopware will automatically try to resolve relative URLs for all your categories and products. These will then be accessible in the `seoUrls` field of each record.
```
[the remark this answers is not in the document]
Comment (sushmangupta) [reply docs/guides/faq.md:35]: If your request includes the `sw-include-seo-urls: 1` header, Shopware will automatically try to resolve relative URLs for all your categories and products. These will then be accessible in the `seoUrls` field of each record.

File: docs/guides/faq.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Christian-Rades) [reply docs/guides/faq.md:17]: Sounds good :+1:

File: docs/guides/faq.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Christian-Rades) [reply docs/guides/faq.md:35]: :heavy_check_mark:

File: docs/guides/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Christian-Rades) [reply docs/guides/faq.md:15]: Do mean like this ?:
> How to check whether a user is currently logged in.

[the remark this answers is not in the document]
Comment (sushmangupta) [reply docs/guides/faq.md:15]: Yes right and with a question mark please. :)

File: docs/guides/quick-start/handling-the-payment.md [hunk 4]
```diff
@@ -81,50 +85,50 @@ In order to provide a more common interface, we advise to use the **optional** p
...
+* Transaction identifier
+* Payment method identifier
+* Finish page URL
+* Error page URL
 
-#### Redirect to payment provider / gateway
+#### Redirect to payment provider/ gateway
 
-All this information will be assembled into a **return URL** for the external payment provider containing the token as a parameter `_sw_payment_token`. Together with this **return URL** \(neither the `finishUrl` nor the `errorUrl` - it's a Shopware endpoint\), Shopware will redirect your call to the external payment endpoint to let it conduct the payment.
+All the above transaction-related information will be assembled into a **return URL** for the external payment provider containing the token as a parameter `_sw_payment_token`. Together with this **return URL** \(neither the `finishUrl` nor the `errorUrl` - it's a Shopware endpoint\), Shopware will redirect your call to the external payment endpoint to let it conduct the payment.
```
Comment (sushmangupta) [inline docs/guides/quick-start/handling-the-payment.md:111]: Please remove this information in brackets : (neither the `finishUrl` nor the `errorUrl` - it's a Shopware endpoint\)

File: docs/guides/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Christian-Rades) [reply docs/guides/faq.md:15]:
```suggestion
**How to check whether a user is currently logged in?**
```

**Comment by sushmangupta** [comment]
All changes have been made.

**Review by Christian-Rades — APPROVED** [review]

**Pull request merged**